modules/command/dc/dc-challenge-check-command.js

```diff
@@ -0,0 +1,84 @@
+const Command = require('../command');
+const models = require('../../../models/index');
```

Comment:
No need to state 'index' it is searched by default.

Reply:
👍